app/views/dashboard/autoyast.xml.erb

```diff
@@ -9,7 +9,12 @@
         <interpreter>shell</interpreter>
         <source><![CDATA[
         #!/bin/sh
-        echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        MINION_CFG="/etc/salt/minion.d/master.conf"
+        [ -f $MINION_CFG ] && MINION_ID=$(grep id $MINION_CFG | sed 's/id: //')
```

Comment:
If the autoyast profile is requested, do we really need to check this? This will only be requested and interpreted in an unattended installation right? So we may just assume this is a clean installation and do the `echoes` without having to check anything else, right?